src/array.cr

```diff
@@ -1430,6 +1430,62 @@ class Array(T)
     self
   end
 
+  # Returns `true` when array starts with *other*, otherwise `false`.
+  #
+  # ```
+  # a = [1, 2, 3]
+  # a.starts_with?([1, 2]) # => true
+  # a.starts_with?([2, 3]) # => false
+  # ```
+  def starts_with?(other : Array)
+    return false if other.size > size
+    self[0, other.size] == other
+  end
+
+  # ditto
+  def starts_with?(other : Enumerable)
+    starts_with?(other.to_a)
+  end
+  
+  # Returns `true` when array starts with *element*, otherwise `false`.
+  #
+  # ```
+  # a = [1, 2, 3]
+  # a.starts_with?(1) # => true
+  # a.starts_with?(2) # => false
+  # ```
+  def starts_with?(el : T)
```

Comment:
Let's not use such confusing abbreviations, please. Also there's `element` in the description.
```suggestion
  def starts_with?(element : T)
```